### ISSUE TITLE
Inline JS (in onclick) is not replaced in AJAX-response 

### ISSUE THREAD

**Issue by andreak** [lift/framework#1700]
Ref this thread
https://groups.google.com/d/msg/liftweb/b6lUWIDOSiY/_6dxm2sDo64J

With the new mechanism in place to replace code like this:
 
```html
<span onclick="cranks()">
```
 
with
 
```html
<span id="lift-event-js-F8991755608893YMP2K">
```
 
it doesn't seem to apply for the same code being served as AJAX-response, meaning the same code will get the original onclick-markup when being served as AJAX.

**Comment by Shadowfiend** [comment]
So this is going to require some fiddlin'.

We'll probably want to move the work that's done in `fixAttrs` in `LiftMerge`
to somewhere in `LiftSession`, where it can in turn be called by `LiftSession`'s
`fixHtml`. We'll then need `fixHtml` to return both the adjusted `NodeSeq` and
the resulting `EventAttribute`s. `LiftMerge` would add those to the page JS, while
`JsCmd`'s `fixHtmlAndJs` would instead append the relevant JS to the `List[JsCmd]`
it already returns.

**Comment by Shadowfiend** [comment]
Going to see about getting started on this today, but wanted to jot down an approach first.

**Comment by fixundfertig123** [comment]
Hi,

thanks for picking up this issue. 

I just wanted to let you know that this new mechanism causes problems with angularJS libs like Angular-xeditable (http://vitalets.github.io/angular-xeditable/), which provides editable elements. The following example shows the problem, where the actual scripts are never called.

**Original HTML code embedded by 'data-lift="embed?what=html':**
```
<span editable-text="list.name" buttons="yes" onbeforesave="XXXXXXXX(list, $data)" onaftersave="YYYYYYYYYY(list)">{{list.name || 'n.a.' }}</span>
```

**Resulting HTML code by using 3.0-M3 (works fine):**
```
<span editable-text="list.name" buttons="yes" onbeforesave="XXXXXXXX(list, $data)" onaftersave="YYYYYYYYYY(list)" class="ng-scope ng-binding editable editable-click">Dummy</span>
```

**Resulting HTML code by using 3.0-M6 (NOT WORKING):**
```
<span editable-text="list.name" buttons="yes" id="lift-event-js-F398556110367JMN31O" class="ng-scope ng-binding editable editable-click">Dummy</span>
```

Since this breaks the usability of my site, is there a way to disable it by a code-fragement?

Cheers and thanks,

Fritz

**Comment by Shadowfiend** [comment]
The above is unrelated to this issue and is instead addressed by issue #1740 .

**Comment by Shadowfiend** [comment]
This sucker's next on my hit list… The oldest unread email in my inbox :p
